client/components/SideImages.jsx

```diff
@@ -4,16 +4,18 @@ function SideImages(props) {
     const items = [];
     for (var i = 1; i < props.numOfImgs + 1; i++) {
         if (i === 2 && props.videoThumb !== null) {
-            var videoThumbnail = props.videoThumb;
             items.push(
-                <img key='video' className='side video' src={videoThumbnail} onClick={props.onVideoClick} />
-            );
+                <span className='videoThumbContainer'>
+                    <img key='video' className='side video' src={props.videoThumb} onClick={props.onVideoClick} />
```

Comment:
keys will generally want to be different for each specific item.  I'd change these and just add a number on to them (the current i or something) to make them unique.

Reply:
Done